akka-http-core/src/main/scala/akka/http/impl/engine/ws/Handshake.scala

```diff
@@ -22,7 +28,8 @@ import scala.reflect.ClassTag
 private[http] object Handshake {
   val CurrentWebsocketVersion = 13
```

Comment:
Since we're passing the `autobahn`... 
I guess I don't need to ask about all the spec rules about "when client wants 7 but you only have 13 you must reply with 13" etc, right?

(just one sanity check, since I expect all these things to be covered in the autobahn suite - won't ping those kinds of questions :-))